Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -6,7 +6,8 @@ import Stripe
 import UIKit
 
 internal protocol AddNewCardViewControllerDelegate: class {
-  func presentAddCardSuccessfulBanner(_ message: String)
+  func addNewCardViewControllerSucceeded(with message: String)
```

Comment:
I'm happy to discuss the naming on these

Reply:
Sounds good. At some point I was passing vc back but then decided not to for obvious reasons...I'll change this back.